sacrebleu/metrics/base.py

```diff
@@ -8,12 +8,12 @@
 import json
 import logging
 import statistics
-from typing import List, Sequence, Any, Optional, Dict
 from abc import ABCMeta, abstractmethod
+from typing import Any, Dict, List, Optional, Sequence
 
-from .. import __version__
+from ..version import __version__
```

Comment:
This import change eliminates a circular dependency between the `sacrebleu` package and this module.